app/assets/stylesheets/custom.scss

```diff
@@ -111,3 +127,73 @@ body {
     }
   }
 }
+
+// sidebar
+aside {
+  section.user_info {
+    margin-top: 20px;
+  }
+
+  section {
+    padding: 10px 0;
+    margin-top: 20px;
+
+    &:first-child {
+      border: 0;
+      padding-top: 0;
+    }
+
+    span {
+      display: block;
+      margin-bottom: 3px;
+      line-height: 1;
+    }
+
+    h1 {
+      font-size: 1.4em;
+      text-align: left;
+      letter-spacing: -1px;
+      margin-bottom: 3px;
+      margin-top: 0px;
+    }
+  }
+}
+
+.gravatar {
+  float: left;
+  margin-right: 10px;
+}
+
+.gravatar_edit {
+  margin-top: 15px;
+}
+
+// forms
+input,
+textarea,
+select,
+.uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
+}
+
+input {
+  height: auto !important;
+}
+
+#error_explanation {
+  color: red;
+  ul {
+    color: red;
+    margin: 0 0 30px 0;
+  }
+}
+
+// .field_with_errors {
```

Comment:
sao lại cmt

Reply:
cái này e làm theo tutorial mà có class "has-error" và $state-danger-text e chưa thấy nó tạo ở đâu nên tạm thời e cmt lại ạ

Reply:
vậy thì thêm //TO-DO a nhé

Reply:
> vậy thì thêm //TO-DO a nhé

ok e :))